programs/ssl/ssl_client2.c

```diff
@@ -1976,7 +1978,7 @@ int main( int argc, char *argv[] )
 #endif /* MBEDTLS_X509_CRT_PARSE_C */
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_ECP_C)
```

Comment:
Do we need the guard by `MBEDTLS_ECP_C`? I see where this originates from - _we_ only offer ECDHE suites for now - but I think it should be removed because TLS 1.3 also supports other key agreements, and we're adding such soon with PQC.

Reply:
I think it should be replace with MBEDTLS_X509_CRT_PARSE_C . Am I right?